src/orchestrator-tracing/src/lib.rs

```diff
@@ -488,29 +489,46 @@ impl NamespacedOrchestrator for NamespacedTracingOrchestrator {
     }
 }
 
-/// Wraps [`Targets`] to provide a [`Display`](fmt::Display) implementation.
-#[derive(Debug, Clone)]
-pub struct SerializableTargets {
-    /// The parsed targets.
-    pub inner: Targets,
-    /// A string representation of `inner`.
-    pub raw: String,
+/// Wraps [`EnvFilter`] to provide a [`Clone`] implementation.
+#[derive(Debug)]
+pub struct CloneableEnvFilter {
+    filter: EnvFilter,
+}
+
+impl CloneableEnvFilter {
+    pub fn inner_ref(&self) -> &EnvFilter {
+        &self.filter
+    }
+
+    pub fn inner(self) -> EnvFilter {
+        self.filter
+    }
+}
+
+impl Clone for CloneableEnvFilter {
+    fn clone(&self) -> Self {
+        // At the time of this implementation, `EnvFilter` does not implement Clone
+        // but is expected, without explicit documentation saying so, to roundtrip
+        // through its Display implementation [1].
+        //
+        // [1]: https://github.com/tokio-rs/tracing/blob/e603c2a254d157a25a7a1fbfd4da46ad7e05f555/tracing-subscriber/src/filter/env/mod.rs#L944-L953
+        let filter = EnvFilter::from_str(&format!("{}", self.filter)).expect("roundtrips");
+        Self { filter }
+    }
```

Comment:
An alternative here is to just store the string we got the env filter from only, and then generate a new envfilter everytime...but not necessary with this link to test. This does need a TODO to fix upstream though!

Reply:
both done!